components/chart/drilldown.md

```diff
@@ -0,0 +1,205 @@
+---
+title: DrillDown
+page_title: DrillDown 
```

Comment:
Add the component name here too.

```suggestion
page_title: DrillDown 
```

Reply:
This is basically a functionality of the component, not a different chart type. We've discussed it and decided to remove the name of the component.

https://github.com/telerik/blazor-docs/pull/1592#discussion_r1301127062

In our other articles related to chart functionalities, we do not include the component name. Should we add it?